source/locale/fr/LC_MESSAGES/releases.po

```diff
@@ -138,7 +138,7 @@ msgstr ""
 "Si un nouveau `hotifx` a été ajouté, vous devrez l'incorporer à votre branche "
 "`release`. Pour savoir comment procéder, vous devrez tout d'abord déterminer "
 "si quelque chose d'autre a changé, vous ne souhaiterez probablement pas "
-"qu'une `feature` terminée après que vous ayez décidé de publier ne soit "
+"qu'une `feature` terminée après que vous avez décidé de publier ne soit "
```

Comment:
I'm not sure this one is correct

Reply:
> À la différence de « avant que », et quoi que l’on pense souvent, « après que » doit être suivi d’un verbe à l’indicatif et non au subjonctif
[Source](https://www.projet-voltaire.fr/regles-orthographe/apres-qu-il-a-ou-apres-qu-il-ait/)